src/ConstructiveSolidGeometry/SurfacePrimitives/Plane.jl

```diff
@@ -9,10 +9,30 @@ Surface primitive describing a two-dimensional flat plane in three-dimensional s
 """
 struct Plane{T} <: AbstractPlanarSurfacePrimitive{T}
     origin::CartesianPoint{T}
-    normal::CartesianVector{T}
+    normal::CartesianPoint{T}
```

Comment:
The `normal` vector of a `Plane` should not be a `CartesianPoint` but remain a `CartesianVector`